src/fsharp/FSharp.Core/printf.fs

```diff
@@ -1267,18 +1267,53 @@ module internal PrintfImpl =
                 let v = get(key.Value)
                 Cache<'T, 'State, 'Residue, 'Result>.last <- (key.Value, v)
                 v
+    
+    let [<Literal>] MAX_BUILDER_SIZE = 360
+    let [<Literal>] MIN_BUILDER_CAPACITY = 128
```

Comment:
Why these concrete numbers? I think comments are needed to explain (or copy / refer the original ones).

Reply:
Move the constants inside `StringBuilderCache` class and align the comment appropriately.